src/components/cylc/gantt/GanttChart.vue

```diff
@@ -0,0 +1,212 @@
+              ],
+              fillColor: jobColours[this.jobs[i].name],
+            })
+          }
+        } else if (this.timingOption === 'run') {
+          for (let i = startIndex; i < endIndex; i++) {
+            data.push({
+              x: this.jobs[i].name,
+              y: [
+                new Date(this.jobs[i].startedTime).getTime(),
+                new Date(this.jobs[i].finishedTime).getTime(),
+              ],
+              fillColor: jobColours[this.jobs[i].name],
+            })
+          }
+        } else if (this.timingOption === 'queue') {
+          for (let i = startIndex; i < endIndex; i++) {
+            data.push({
+              x: this.jobs[i].name,
+              y: [
+                new Date(this.jobs[i].submittedTime).getTime(),
+                new Date(this.jobs[i].startedTime).getTime(),
+              ],
+              fillColor: jobColours[this.jobs[i].name],
+            })
+          }
+        }
+      }
+      return [{ data }]
+    },
+
+    chartOptions () {
+      return {
+        chart: {
+          animations: {
+            enabled: this.$store.state.app.reducedAnimation ? false : this.animate,
+            easing: 'easeinout',
+            speed: 300,
+            animateGradually: {
+              enabled: true,
+              delay: 150,
+            },
+            dynamicAnimation: {
+              enabled: true,
+              speed: 350,
+            },
+          },
+          fontFamily: 'inherit',
+          toolbar: {
+            tools: {
+              download: `<svg class="w-100 h-100"><path d="${mdiDownload}"></path></svg>`,
+            },
+          },
+        },
+        tooltip: {
+          custom: function ({ seriesIndex, dataPointIndex, w }) {
+            const startTime = new Date(w.config.series[seriesIndex].data[dataPointIndex].y[0])
+            const finishTime = new Date(w.config.series[seriesIndex].data[dataPointIndex].y[1])
+            return (
+              '<div class="apexcharts-tooltip-candlestick">' +
+              '<div>Start: <span class="value">' +
+              startTime +
+              '</span></div>' +
+              '<div>Finish: <span class="value">' +
+              finishTime +
+              '</span></div>' +
+              '</div>'
+            )
+          },
+        },
+        plotOptions: {
+          bar: {
+            horizontal: true,
+          },
+        },
+        xaxis: {
+          title: {
+            text: 'Time',
+          },
+          type: 'datetime',
+        },
+      }
+    },
+  },
+
+  icons: {
+    sortAsc: mdiSortReverseVariant,
+    sortDesc: mdiSortVariant,
+  },
+  itemsPerPageOptions: [
+    { value: 10, title: '10' },
+    { value: 20, title: '20' },
+    { value: 50, title: '50' },
+    { value: 100, title: '100' },
+    { value: 200, title: '200' },
+    { value: -1, title: 'All' }
+  ],
+}
+</script>
+
+<style>
+.apexcharts-text {
+  font-size: 1.1rem;
```

Comment:
For consistency with the box & whiskers view?
```suggestion
  font-size: 0.9rem;
```